invenio_vocabularies/records/systemfields/relations.py

```diff
@@ -35,8 +35,9 @@ def _fields(self):
         relations = {}
         for cf in custom_fields:
             if getattr(cf, "relation_cls", None):
-                relations[cf.name] = cf.relation_cls(
-                    f"custom_fields.{cf.name}",
+                rel_name = cf.name.replace(":", "_")  # for namespaces
+                relations[rel_name] = cf.relation_cls(
```

Comment:
I guess this is for being able to `.`-access fields without having to use `getattr`? Not sure how often it's needed but it's probably good for debugging in any case.

Reply:
Yes, is it would be slightly inconsistent naming wise (i.e. json attr vs programatic attr). But with the `:` we could not `.` access the field.

Reply:
I had ended up to the same thing in the past but I kept it like that so we are consistent. Otherwise, it will create confusion....